build.gradle

```diff
@@ -101,13 +101,17 @@ ext.jmxetricsVersion = "1.0.8"
 ext.hadoopVersion = "2.7.2"
 
 dependencies {
-  compile ("org.apache.hadoop:hadoop-client:${hadoopVersion}") {
+
+  /* Note this jar is fetched from our private Maven repo (patched for HADOOP-12807) */
+  compile ("org.apache.hadoop:hadoop-aws:${hadoopVersion}") {
+    exclude group: "com.sun.jersey"
+  }
+
+  compile ("org.apache.hadoop:hadoop-common:${hadoopVersion}") {
```

Comment:
I'm actually not sure how this compiles and runs tests without the `hadoop-client` dependency, since it doesn't appear that `hadoop-common` pulls in the YARN and HDFS client libs we need. In any case,  wouldn't it be better to leave the explicit dependency on `hadoop-client` and `hadoop-aws` (since those are the libraries we actually consume), and exclude `com.sun.jersey` just from `hadoop-common`?